apps/antidote/src/antidote.erl

```diff
@@ -56,6 +56,9 @@
     get_txn_property/2
 ]).
 
+-export_type([snapshot_time/0, dcid/0, clock_time/0,
```

Comment:
Agreed, should eventually refactor the header file accordingly. I just would suggest to keep it in one place for the moment to avoid confusion. E.g. is `antidote:dcid()` the same as `dcid()`???

Reply:
Well I guess the question that you probably ask here - wherever or not `antidote:dcid()` is treated in the same way as `dcid()` by tools like dialyzer. And honestly I do not know definite answer for that, so I try to use fully-qualified names to be on the safer side. 

But also, defined type like this could have different definitions in different modules. So when I see the `antidote:dcid()` type in multiple specs from different files I have a guaranty without checking other file definitions or hrl files, that it's the same type in both cases. While just `dcid()` without module that exports it could have different definitions. So in order to know - I need to check definition. Maybe for `dcid()` it's not that likely to end up in that situation, but some more general type like `key()` it could very well be the case.

Reply:
Sorry, I meant for this very specific `antidote:dcid()`: Is it the same thing as what is currently called `dcid()`?

Reply:
Ahh, yes, I didn't change `dcid()`, just exported it.

Reply:
My bad - need more coffee first, before I read code...